koschei/frontend/views.py

```diff
@@ -920,3 +922,16 @@ def package_state(row):
     return render_template("affected-by.html", package_state=package_state,
                            dep_name=dep_name, evr1=evr1, evr2=evr2,
                            collection=collection, failed=failed)
+
+
+@app.route('/stats')
+@stats_tab.master
+def statistics():
+    now = db.query(func.now().label('now')).one().now
+    scalar_stats = db.query(ScalarStats).one()
```

Comment:
Can you add some handling for the situation where stats weren't generated yet? Just so it doesn't crash. Like a redirect with flash or something.